src/main/java/frc/robot/UserInterface.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Team 4909, Bionics
+ * Billerica Memorial High School
+ *
+ * Copyright:
+ *   2021 Bionics
+ *
+ * License:
+ *   MIT: https://opensource.org/licenses/MIT
+ *   See the LICENSE file in the project's top-level directory for details.
+ */
+
+package frc.robot;
+
+import java.lang.Object;
+import java.util.HashMap;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import frc.peyton.Drivetrain;
+import frc.bionic.swerve.command.DriveWithJoystick;
+
+public class UserInterface
```

Comment:
I wonder if we should adopt the RobotContainer pattern that Wpilib is recommending: https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/schedulereventlogging/RobotContainer.java#L21-L26